OidcDebugger/Views/Home/Index.cshtml

```diff
@@ -1,197 +1,246 @@
+                    <div class="md-form">
+                        <label for="responseType" class="form-group-label">
+                            Response type <span class="required-text">(required)</span>
+                        </label>
+                        <div class="form-group-inline">
+                            <div class="form-group">
+                                <input type="checkbox" id="responseType-code" value="code"
+                                       v-model="responseTypesArray" v-on:focus="showInfo" v-on:click="saveParameters" required>
+                                <label for="responseType-code" class="select-label">code</label>
+                            </div>
+                            <div class="form-group">
+                                <input type="checkbox" id="responseType-token" value="token"
+                                       v-model="responseTypesArray" v-on:focus="showInfo" v-on:click="saveParameters" required>
+                                <label for="responseType-token" class="select-label">token</label>
+                            </div>
+                            @if (Tenant.Shortname == "oidc")
+                            {
+                                <div class="form-group">
+                                    <input type="checkbox" id="responseType-id_token" value="id_token"
+                                           v-model="responseTypesArray" v-on:focus="showInfo" v-on:click="saveParameters" required>
+                                    <label for="responseType-id_token" class="select-label">id_token</label>
+                                </div>
+                            }
+                        </div>
+
+                        <div v-show="responseTypesArray.indexOf('code') > -1">
+                            <label for="pkceMethod" class="form-group-label">
+                                PKCE Method
+                            </label>
+
+                            <div class="md-form">
+                                <input v-on:change="saveParameters" v-model="pkceMethod" type="radio" id="pkce-sha256" name="pkce-method" value="S256" />
+                                <label for="pkce-sha256" class="select-label">SHA-256</label>
+
+                            </div>
+                            <div class="md-form">
+                                <input v-on:change="saveParameters" v-model="pkceMethod" type="radio" id="pkce-plain" name="pkce-method" value="plain" />
+                                <label for="pkce-plain" class="select-label">plain</label>
+
+                            </div>
+                            <div class="md-form">
+                                <input v-on:change="saveParameters" v-model="pkceMethod" type="radio" id="pkce-disabled" name="pkce-method" value="disabled" />
+                                <label for="pkce-disabled" class="select-label">disabled</label>
+                            </div>
```

Comment:
nit: this radio should be displayed horizontally to match the other checkboxes on the page

Reply:
@bdemers [87da64b](https://github.com/nickolasfisher/oidc-debugger/commit/87da64b43a4ca41fb5afa6cadb0ebf65b600d075)